app/src/main/java/org/oppia/android/app/profile/AdminAuthActivityPresenter.kt

```diff
@@ -47,28 +46,24 @@ class AdminAuthActivityPresenter @Inject constructor(
 
     setTitleAndSubTitle(binding)
 
-    binding.adminAuthInputPin.addTextChangedListener(object : TextWatcher {
-      override fun onTextChanged(confirmPin: CharSequence?, start: Int, before: Int, count: Int) {
-        confirmPin?.let {
-          authViewModel.errorMessage.set("")
-        }
+    // [onTextChanged] is a extension function defined at [TextInputEditTextHelper]
+    binding.adminAuthInputPinEditText.onTextChanged { comfirmPin ->
```

Comment:
Just a small typo here. Should be `confirmPin`.

Reply:
nice catch